node/cli/src/command.rs

```diff
@@ -75,6 +80,9 @@ fn load_spec(id: &str) -> std::result::Result<Box<dyn ChainSpec>, String> {
 		#[cfg(feature = "frequency-local")]
 		"frequency-paseo-local" =>
 			Ok(Box::new(chain_spec::frequency_paseo_local::local_paseo_testnet_config())),
+		#[cfg(feature = "frequency-bridging")]
```

Comment:
Perhaps this was for later, but I thought we were going to add a different feature flag for the spec vs the actual bridging code

Reply:
Or I guess here it should be this right?

```suggestion
		#[cfg(feature = "frequency-local")]
```

Reply:
Yes, thanks for catching this, we plan on revisiting this as we add `frequency-westend-testnet`.